pingpong/lti/server.py

```
@@ -1641,3 +1882,238 @@ async def link_lti_group(
+            status_code=500, detail="Failed to sign Deep Linking response"
+        ) from e
+
+    state["completed_at"] = now.isoformat()
+    oidc_session.extra = json.dumps(state)
+    request.state["db"].add(oidc_session)
+    await request.state["db"].flush()
+    return LTIDeepLinkCompleteResponse(
+        deep_link_return_url=state["deep_link_return_url"],
+        jwt=response_jwt,
+    )
+    (LTI_CLAIM_MESSAGE_TYPE_KEY,)
```

Ruff (0.16.1)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)